xcpmd/src/prototypes.h

```diff
@@ -20,7 +20,6 @@
 
 /* acpi-events.c */
 int xcpmd_process_input(int input_value);
-void adjust_brightness(int increase, int force);
```

Comment:
Was this intentional? Is there a function definition that needs to be removed as well?

Reply:
```adjust_brightness``` the function was removed by Eric in 6c2abf4314a033259f26d27a701f129841c108dd, he forgot to remove the function definition so this is just following through on the rest of the change.